src/services/VerifyAccountRequestProcessor.ts

```diff
@@ -143,20 +143,20 @@ export class VerifyAccountRequestProcessor {
   			evidence: [
   				{
   					txn: verifyResponse?.expRequestId,
+  					attemptNum: session.attemptCount ?? 1,
   				},
   			],
   		},
   		restricted: {
-  			Experian_request_details: [
-  				{
-  					name:`${givenName} ${surname}`,
-  					sortCode: verifyAccountPayload?.sort_code,
-  					accountNumber: verifyAccountPayload?.account_number,
-  					attemptNum: session.attemptCount ?? 1,
-  				},
-  			],
+  			name: person.name,
```

Comment:
Will this not also return the middlenames?

Reply:
Yep, it returns the entire name object we receive in `shared_claims` which includes all 'GivenNames', this is in line with the [event catalogue](https://event-catalogue.internal.account.gov.uk/events/BAV_EXPERIAN_REQUEST_SENT/)

Reply:
This needs to reflect what we send to Experian so should exclude the middle names